### ISSUE TITLE
two working directories exists 

### ISSUE THREAD

**Issue by iva-nova-e-katerina** [Significant-Gravitas/Auto-GPT#1923]
### Duplicates

- [X] I have searched the existing issues

### Steps to reproduce 🕹

...
EXECUTE_LOCAL_COMMANDS=True
...
python -m autogpt --continuous
y

### Current behavior 😯

/home/user/Auto-GPT/auto_gpt_workspace/debt_repayment.py exist
/home/user/Auto-GPT/autogpt/auto_gpt_workspace/debt_repayment.py does **NOT** exist

```
NEXT ACTION:  COMMAND = write_to_file ARGUMENTS = {'file': 'debt_repayment.py', 'text': "debts = {\\\\n 'Credit Card': {'balance': 5000, 'interest_rate': 0.15},\\\\n 'Auto Loan': {'balance': 10000, 'interest_rate': 0.05},\\\\n 'Student Loan': {'balance': 20000, 'interest_rate': 0.03}\\\\n}\\\\n\\\\nmonths = 12\\\\n\\\\nfor i in range(months):\\\\n total_interest = 0\\\\n total_payment = 0\\\\n\\\\n for debt in debts:\\\\n balance = debts[debt]['balance']\\\\n interest_rate = debts[debt]['interest_rate']\\\\n interest_charge = balance * interest_rate / 12\\\\n principle_payment = balance / months\\\\n\\\\n balance += interest_charge - principle_payment\\\\n balance = max(balance, 0)\\\\n debts[debt]['balance'] = balance\\\\n\\\\n total_interest += interest_charge\\\\n total_payment += principle_payment\\\\n\\\\n print(f'Month {i + 1}: Total payment - {total_payment}, Total interest - {total_interest}')"}
SYSTEM:  Command write_to_file returned: Error: File has already been updated.
 THOUGHTS:  None
REASONING:  None
CRITICISM:  None
SPEAK:  None
Attempting to fix JSON by finding outermost brackets 
Apparently json was fixed. 
NEXT ACTION:  COMMAND = execute_python_file ARGUMENTS = {'file': 'debt_repayment.py'}
Executing file 'debt_repayment.py' in workspace '/home/user/Auto-GPT/autogpt/auto_gpt_workspace'
SYSTEM:  Command execute_python_file returned: Error: File 'debt_repayment.py' does not exist.

```

### Expected behavior 🤔

common working directory for all files

### Your prompt 📝

```yaml
# Paste your prompt here
```

**Comment by Imccccc** [comment]
where those file located in your computer?
I believe you have to tell auto gpt that the file fullpath and the goal you want to achive.

**Comment by aSocialMenace** [comment]
I've seen the same bug. I reinstalled autogpt and it went away

**Comment by robnraz** [comment]
Happened to me as well several times. 
The repository/code adds a 2nd \autogpt\ folder on its own

I have also seen on Windows how the bot tries to find a file in /home/ubuntu/xxx/xxx file structure instead of the /Auto-GPT/auto_gpt_workspace/ structure

**Comment by Pwuts** [comment]
I think this is fixed in #1947. Please reopen if not.
